Orange/widgets/unsupervised/owhierarchicalclustering.py

```diff
@@ -763,6 +763,9 @@ class Outputs:
     cluster_roles = ["Attribute", "Class variable", "Meta variable"]
     basic_annotations = ["None", "Enumeration"]
 
+    class Error(widget.OWWidget.Error):
+        not_finite_distances = Msg("Not all distances are finite")
```

Comment:
Good point. No verb, no dot. Verb, dot. Period. :)

However, some error messages are not sentences, like "Out of memory" or "Error during optimization". What about those?

Message without verb bad, message with verb good? Makes them longer, though. Sometimes long not good, when widget narrow.